libs/permission-checker/src/Check/Ai/CanCreateAutomations.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker\Check\Ai;
+
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+use Keboola\PermissionChecker\Feature;
+use Keboola\PermissionChecker\PermissionCheckInterface;
+use Keboola\PermissionChecker\Role;
+use Keboola\PermissionChecker\StorageApiToken;
+
+class CanCreateAutomations implements PermissionCheckInterface
```

Comment:
Co Guest role? Ten nemůže sahat na orchestrace pokud to správně chápu. 

https://help.keboola.com/management/project/users/#user-roles

Reply:
nemůže dělat schedules (kvůli tomu, že nemůže zakládat tokeny), flows editovat může, zběžně jsem to s guestem proklikal a fungovalo to, tak jsem ho tu nechal

Reply:
°Ještě mě napadá, jestli by se to nemělo jmenovat "can modify automations" spíš ;)

Reply:
Jo, to je asi lepší název